azure-pipelines/build-release.yml

```diff
@@ -94,7 +94,7 @@ stages:
       - template: steps/install-development-dependencies.yml
 
       - bash: |
-         echo "##vso[task.setvariable variable=temp_docs_path]`get-config --key DOCUMENTATION_TEMP_CI_OUTPUT_PATH`"
+         echo "##vso[task.setvariable variable=temp_docs_path]`get-config --key DOCUMENTATION_PRODUCTION_OUTPUT_PATH`"
```

Comment:
Why do we need to set `temp_docs_path`? 🤔 Can we not do `$(get_config --key BLAH)` on the line 101?

Reply:
We can, I have changed it 👍

Reply:
No wait, sorry I did think about this but the weekend had blatted my memory - in Azure the `$()` notation is used to access variables not run a command so changing it to `` but this notation is bash-specific so its safer to run it in a bash command and then use the variable later on.